src/askscript/__tests__/01-basics/program03d-int.askcode

```diff
@@ -1 +1 @@
-ask(123,-456)
\ No newline at end of file
+ask(123,456)
```

Comment:
What happened to the minus? 🤔

Reply:
I will try to figure it out, but if it takes too long, how about you help me?

Reply:
Sure, let's figure this out together 🙌

Reply:
What I discovered so far is it has to do with Prettier:
<img width="1097" alt="Screen Shot 2020-07-02 at 12 46 51" src="https://user-images.githubusercontent.com/12752520/86354904-1bf7c780-bc6a-11ea-83ab-6b8a9fb0ad05.png">

Let me do more research.

Reply:
@czerwinskilukasz1 yes, that makes sense. Prettier needs to be aware of new constructs we're adding to be able to produce correctly formatted code

Reply:
And .askcode is currently reflection of the formatted code to make sure that prettier layer also gets tested

Reply:
The issue is caused by the assumption that all operators are binary.  So `join` expects 2 arguments and wants to add an operator between them here:
![image](https://user-images.githubusercontent.com/12752520/86356890-50b94e00-bc6d-11ea-9927-c7fe0d5f55eb.png)
However, for unary operators there is obviously only one operand.

Reply:
Fixed